lib/python/picongpu/plugins/data/energy_histogram.py

```diff
@@ -173,8 +174,8 @@ def _get_for_iteration(self, iteration, species, species_filter="all",
         if not include_overflow:
             del data['underflow']
             del data['overflow']
-
+        dt = FindTime(self.run_directory).get_dt()
         if len(iteration) > 1:
-            return data.loc[iteration].values, bins
+            return data.loc[iteration].values, bins, iteration, dt
         else:
             return data.loc[iteration].values[0, :], bins
```

Comment:
Also please make sure that the same kind of data is returned in the `if` and the `else` branches.

Reply:
@kossag14 this still needs to be addressed